spinta/client.py

```diff
@@ -19,7 +19,9 @@ class RemoteClientCredentials:
     secret: str         # client secret
     server: str         # server URL
     remote: str         # remote name given in credentials.cfg section
-    scopes: List[str]   # list of client scopes
+    scopes: List[str]   # remote name given in credentials.cfg section
```

Comment:
```suggestion
    scopes: str | None   # remote name given in credentials.cfg section
    resourrce_server: str | None
    client_id: str | None
```

Reply:
Šis laukas privalomas.

P.S. Blogas komentaras kodo eiluteje, pataisysiu.